### PR TITLE
Remove flaky inventory tests

### PR DESCRIPTION
**Pull request by cpg1111**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -98,5 +98,7 @@ commands =
     ansible-test sanity --local
 
 [testenv:integration]
+setenv = 
+  HOME = $HOME
```
Comment (albertodonato) [inline tox.ini:102]: you can use just 
```
passenv = 
    HOME
```

**Review by albertodonato — COMMENTED** [review]
I'm wondering if it would make sense to keep the test for the ready status, and just drop the commissioning one.

That should guarantee that machines are ready to be used by following tests.

Thoughts?

**Comment by cpg1111** [comment]
> I'm wondering if it would make sense to keep the test for the ready status, and just drop the commissioning one.
> 
> That should guarantee that machines are ready to be used by following tests.
> 
> Thoughts?

Well so two issues around that, the first being the ready status tests have the reverse issue as the commissioning ones it turns out, where if the tests execute too fast, they will fail. And the second issue is each test module deletes and creates its own machine(s) anyway.

**Comment by albertodonato** [comment]
> Well so two issues around that, the first being the ready status tests have the reverse issue as the commissioning ones it turns out, where if the tests execute too fast, they will fail. And the second issue is each test module deletes and creates its own machine(s) anyway.

I see, then +1 on removing them.

**Review by albertodonato — APPROVED** [review]

**Pull request merged**